CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased] (beta)
 ### Added
 
+- New Flag `--on-error` which controls recoverable error handling behavior.  Accepts `--on-error fail` to fail out immediately, and `--on-error continue` (default) to proceed with the operation.
```

Comment:
It is.  I've been informed that forcing- or even defaulting to- fail fast isn't an acceptable system state.  There are other ways around the issue in the short term (ie: forcing hard failures in necessary locations).  But at large, this is what we're sticking with.

Reply:
then before merge this we need to be certain we're forcing hard failures when we need to. Otherwise we could end up with backups being used as bases for incrementals when they shouldn't be

Reply:
I'm not aware of all those critical points of failure.  Could I get your help with that?